integration_test/ops_agent_test.go

```diff
@@ -1441,9 +1443,48 @@ func TestTCPLog(t *testing.T) {
 			t.Fatal(err)
 		}
 
+		command := "echo '{\"msg\":\"test tcp log 1\"}{\"msg\":\"test tcp log 2\"}' > /dev/tcp/localhost/5170 && echo '{\"msg\":\"test tcp log 3\"}{\"msg\":\"test tcp log 4\"}' > /dev/tcp/localhost/5170"
+		// Installing Netcat or equivalent tools on Windows is tricky so we can instead
+		// use this Powershell Script that can send TCP messages.
+		if gce.IsWindows(platform) {
```

Comment:
Would also be nice to have the linux part do the same tbh

Reply:
I do agree, fluent bit is better than the TCP script. I changed the Linux test to use fluent bit. 

For Windows, the only way we could use fluent bit to send TCP requests is if we use stdin or tail input plugins. Stdin isnt supported in Windows and tail, in my experience has been flaky or at least quite tricky to get right as it seems to take a few commands for fluent bit to actually pick it up. 

The reason why we can't use somethign like 'random' or 'dummy' like we did in fluent forward test is because we want to query with actual log id to test that data from multiple requests are going through. With 'dummy' the data is always the same and with random the data is random values so querying is hard.

Reply:
https://docs.fluentbit.io/manual/pipeline/inputs/dummy#configuration-parameters

Oh I thought there was a `dummy` config field that would let you configure what message to send. We could have 2 receivers with different messages, and then output it all to the same tcp destination. I don't feel too strongly about this tho, so if its too much trouble maybe its not worth it. 

deferring to @braydonk and @shafinsiddique for what is best here

Reply:
For other receivers, we often test with 1 log. In this case, it sounds like that makes it much simpler too. Is there a specific reason we need to test several tcp payloads here in this test? Has it got something to do with the `in_tcp` bug we found?

Reply:
Having multiple messages will test fluent-bit's ability to separate independent JSON chunks into records when there's no newline in-between.

Reply:
yep, the in_tcp bug arises only on second emitted logs. The first ones go through. That's why we're adding an additional log

Reply:
> Having multiple messages will test fluent-bit's ability to separate independent JSON chunks into records when there's no newline in-between.

this i believe you're talking about why we send two jsons per message. Ridwan i believe was asking why we send 2 messages (using the && or -and)